tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Kafka/OffsetsCommittedCallbacks.cs

```diff
@@ -0,0 +1,47 @@
+// <copyright file="OffsetsCommittedCallbacks.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+#nullable enable
+
+using System;
+using Datadog.Trace.DuckTyping;
+using Datadog.Trace.Util.Delegates;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Kafka
+{
+    internal readonly struct OffsetsCommittedCallbacks : IBegin2Callbacks, IVoidReturnCallback
```

Comment:
@tonyredondo, could you review this instrumentation please (and its usages), as it's the first in-the-wild usecase for the new delegate instrumentation 😃

Reply:
LGTM, I left a comment about getting the type directly from the ducktype proxy.